### ISSUE TITLE
Reunião 002 - 2019-09-02

### ISSUE THREAD

**Issue by antonioribeiro** [convergencias/selo-convergente#5]
![image](https://user-images.githubusercontent.com/3182864/63639098-35051480-c666-11e9-82d7-efc58193fb0c.png)

Caras e Caros Convergentes,
Na reunião de quarta-feira passada ficou acertado que nossos encontros ocorrerão em dias alternados da semana para que aqueles que têm compromissos em determinados dias possam participar de algumas reuniões.
O Antonio Carlos Ribeiro conseguiu o luxo de utilizarmos uma das salas de Comissões da ALERJ às segundas feiras, desta forma podemos confirmar o próximo encontro do Selo Convergente no dia 02 de Setembro às 19hs?
A pauta será para pensarmos na estruturação do Selo (COMO?)

Quem puder, favor colocar o nome na lista abaixo para termos uma ideia do quórum:

- [ ] Claudio Coutada
- [ ] Antonio Carlos Ribeiro
- [ ] David Nemer (online)
- [ ] Leandro Correia (online)
- [ ] Vinicius Peçanha (online)
- [ ] Juliet Matos
- [ ] André Luiz
- [ ] Guilherme J.
- [ ] Lucas Aunton (online)
- [ ] Carolina Aragão (online)
- [ ] Waltinho
- [ ] Orlando Thomé
- [ ] João Luiz Ferreira Costa

**Comment by walter1811** [comment]
Sugestão de pauta:
- decidir nossa estratégia (missão, visão, valores, por que, como, o que)
- montar um canvas
- marcar um próximo encontro pra decidir nossos OKRs pros próximos 90 dias)

**Comment by orlandothome** [comment]
Concordo com a sugestão de pauta, mas creio que na reunião do dia 2/9 devamos esgotar o 1º ponto, deixando os outros dois para a próxima reunião.

**Comment by J943** [comment]
Acho a ideia de focar no 1 topico fundamental.
Nao acho que uma candidatura unica prefeitura selo convergente deva estar na nossa missao.
Candidaturas podem ser objetivos.
Candidatura unica pode estar no horizonte ou na visao.
PS - devo me atrasar porque 19 hs é passagem de plantao no hospital
